src/main/scala/monocly/PIso.scala

```diff
@@ -5,13 +5,12 @@ import monocly.impl._
 import monocly.internal.Applicative
 import monocly.functions.Index
 
-type PIso[-S, +T, +A, -B] = POptic[GetOne & ReverseGet, S, T, A, B]
+type PIso[-S, +T, +A, -B] = POptic[OpticCan.Get & OpticCan.ReverseGet, S, T, A, B]
```

Comment:
I like zio-prelude approach with a `classic` object to map traditional names with new ones.

https://github.com/zio/zio-prelude/blob/06c4af0aafbbce76c8e7a204f1ed2ec414c94727/core/shared/src/main/scala/zio/prelude/package.scala#L95-L107